web/js/searchable-option-list-2.0.4.js

```diff
@@ -1037,10 +1037,11 @@
                 if ((this.config.multiple || this.config.allowNullSelection) && !$changedItem.prop('disabled')) {
                     $('<span class="sol-quick-delete"/>')
                         .html(this.config.texts.quickDelete)
-                        .click(function () {
+                        .click(function () { // deselect the project and refresh the search
```

Comment:
Add there this note
```javascript
/*
 * Modified 2017
 */
```

Reply:
1. The page is refreshed – would you like me to add there a check if we are on the main page and then ignore the form submit (or do you have something different in mind)?
2. Project picker is not affected (since I thought that reloading page while selecting/deselecting in project picker would not be very wise)
3. --||--

Reply:
> add there a check if we are on the main page and then ignore the form submit

I think this is preferable. Consider a scenario when the user fills the form with something and then cares about the projects and a removal takes him to search results immediately. That doesn't seem intuitive.

I think that I added a class to body to reflect the current page name so you can maybe use that.

Reply:
2. and 3. Ok